autogen/agentchat/conversable_agent.py

```diff
@@ -731,6 +738,148 @@ async def a_initiate_chat(
+        else:
+            return extracted_response
+
+    def initiate_chats(self, chat_queue: List[Dict]):
+        """Initiate chats with multiple agents.
+
+        Args:
+            chat_queue (List[Dict]): a list of dictionaries containing the information of the chats.
+                Each dictionary should contain the following fields:
+                - "recipient": the recipient agent.
+                - "context": any context information.
+                    "message" needs to be provided if the `generate_init_message` method is not overridden.
+                          Otherwise, input() will be called to get the initial message.
+                    "takeaway_method" can be used to specify the method to extract the takeaway from the chat.
+                        Supported methods are "last_msg" and "reflection_with_llm".
+                        when set "last_msg", it returns the last message of the dialog as the takeaway.
+                        when set "reflection_with_llm", it returns the takeaway extracted using an llm client.
+                        "reflection_with_llm" requires the llm_config to be set in either the sender or the recipient.
+                    "takeaway_prompt" can be used to specify the prompt used to extract the takeaway when takeaway_method is "reflection_with_llm".
+                        Default is None and the following default prompt will be used when "takeaway_method" is set to "reflection_with_llm":
+                        "Identify and extract the final solution to the originally asked question based on the conversation."
+                    "carryover" can be used to specify the carryover information to be passed to this chat.
+                        If provided, we will combine this carryover with the "message" content when generating the initial chat
+                        message in `generate_init_message`. Please override the `generate_init_message` if you want to customize
+                        how to use the carryover information shall be used.
+
+        """
+        receipts_set = set()
+        for chat_info in chat_queue:
+            assert "recipient" in chat_info, "recipient must be provided."
+            receipts_set.add(chat_info["recipient"])
+        assert len(receipts_set) == len(chat_queue), "recipients must be different."
+
+        self._chat_queue = chat_queue
+        self._finished_chats = {}
+        takeaway = []
+        while self._chat_queue:
+            chat_info = self._chat_queue.pop(0)
+            _chat_carryover = chat_info.get("carryover", [])
+            if isinstance(_chat_carryover, str):
+                _chat_carryover = [_chat_carryover]
+            chat_info["carryover"] = _chat_carryover + list(self._finished_chats.values())
+            if "message" not in chat_info:
+                warnings.warn(
+                    "message is not provided in chat_info. input() will be called to get the initial message.",
```

Comment:
minor comment.

chat_info seems to be an internal variable, so the user might not understand this error message.
Consider "message is not provided in a chat_queue entry. input() ..."